### ISSUE TITLE
Current version of JupyterLab not supported.

### ISSUE THREAD

**Issue by cher1998** [altair-viz/jupyterlab_voyager#82]
when ran this in the console
`jupyter labextension install jupyterlab_voyager`

got this error 
`An error occured.
ValueError: The extension "jupyterlab_voyager" does not yet support the current version of JupyterLab.

Conflicting Dependencies:
JupyterLab                        Extension      Package
>=2.0.2 <2.1.0                    >=1.0.3 <2.0.0 @jupyterlab/application
>=2.0.2 <2.1.0                    >=1.0.3 <2.0.0 @jupyterlab/apputils
>=2.0.2 <2.1.0                    >=1.0.3 <2.0.0 @jupyterlab/console
>=4.0.2 <4.1.0                    >=3.0.0 <4.0.0 @jupyterlab/coreutils
>=2.0.2 <2.1.0                    >=1.0.3 <2.0.0 @jupyterlab/docmanager
>=2.0.2 <2.1.0                    >=1.0.4 <2.0.0 @jupyterlab/filebrowser
>=2.0.2 <2.1.0                    >=1.0.3 <2.0.0 @jupyterlab/notebook
>=2.0.2 <2.1.0                    >=1.0.3 <2.0.0 @jupyterlab/rendermime
>=2.0.1 <2.1.0                    >=1.3.0 <2.0.0 @jupyterlab/rendermime-interfaces
See the log file for details:  C:\Users\CHITIR~1.ESW\AppData\Local\Temp\jupyterlab-debug-lt24c2m4.log`

-- JupyterLab version -- 2.0.1

**Comment by rragundez** [comment]
Any update on this one?

**Comment by dreme** [comment]
I'm also getting dependency conflicts after just recently reinstalling anaconda

```
Conflicting Dependencies:
JupyterLabExtension        Package
>=16.8.4 <16.9.0>=16.9.0 <17.0.0 react
>=16.8.4 <16.9.0>=16.9.0 <17.0.0 react-dom
```

**Comment by satemmanuel** [comment]
jupyterlab_voyager" does not yet support the current version of JupyterLab

**Comment by Symbolics** [comment]
Is there a roadmap or estimated time frame for support?

**Comment by ghost** [comment]
TL;DR- we need datavoyager upstream to be ported to react 16.

The 'Conflicting Dependencies:' text comes from some checks in `jupyterlab/commands.py` prior to invoking yarn to build the extension. Jupyterlab 1.1 appears to have changed its approach to core dependencies, they're now listed as resolutions and singletons.

That's a problem for voyager because jupyterlab is using react 16, but the upstream datavoyager project is still on react 15.
There is an outstanding issue for them to address: https://github.com/vega/voyager/issues/843
Progress on that appears to have stalled though.

If you bypass those build checks to get voyager installed anyway, you're likely to see the following JS error (in chrome at least):

```
index.out.js:1439 TypeError: Cannot read property 'oneOfType' of undefined
    at Object.<anonymous> (lib-voyager.js:1)
    at t (lib-voyager.js:1)
```

I've tried forcing the singleton react-dom to 15.7.0 used by upstream but failed to get react-dom to load in jupyterlab 1.1.  I assume that's due to react 16 changes in jupyterlab, but I haven't bottomed that out.

**Comment by ybettign** [comment]
any news on this?

**Comment by Symbolics** [comment]
It looks like it's officially dead: https://github.com/vega/voyager/issues/857

**Comment by smturro2** [comment]
Awesome project! Sad to see it can no longer be supported. I think a disclaimer on the readme would be beneficial for anyone coming across this on the readme

**Comment by ventilator** [comment]
Are there any alternatives available to edit vega-lite plots locally with a GUI?
